SpatialGDK/Source/SpatialGDKServices/Private/SpatialGDKServicesModule.cpp

```diff
@@ -11,9 +11,17 @@ IMPLEMENT_MODULE(FSpatialGDKServicesModule, SpatialGDKServices);
 
 void FSpatialGDKServicesModule::StartupModule()
 {
+	// Create an instance of the local deployment manager for tracking and controlling local deployment status.
+	LocalDeploymentManager = FLocalDeploymentManager();
 }
+
 void FSpatialGDKServicesModule::ShutdownModule()
 {
 }
 
+FLocalDeploymentManager* FSpatialGDKServicesModule::GetLocalDeploymentManager()
```

Comment:
Can this be null? If not lets make it a reference.

Reply:
It should never be null